docs/usage/api.rst

```diff
@@ -62,6 +62,7 @@ Quality control
     :toctree: preprocessing
 
     preprocessing.calculate_qc_metrics
+    preprocessing.display_qc_metrics
```

Comment:
Should this be in the plotting part?

Maybe pp.qc_metrics and pl.qc_metrics?

Reply:
(I'd suggest to remove the "calculate" and "display" as well then.

e.g. umap is tl.umap and pl.umap

Must be reflected in the tutorials as well.